src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -64,9 +63,6 @@ const NavHeader = (props) => {
 
   const AnimatedBlurView = Animated.createAnimatedComponent(BlurView);
 
-  const TopicDefaultIcon =
-    followType === 'incognito' ? TopicAnonDefaultIcon : TopicSignedDefaultIcon;
-
   return (
     <View>
       <View style={{}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `TopicDefaultIcon` constant is removed as it's no longer needed due to the changes in the image rendering logic. This change is consistent with the removal of the imports for `TopicSignedDefaultIcon` and `TopicAnonDefaultIcon`.
```diff
- const TopicDefaultIcon =
-   followType === 'incognito' ? TopicAnonDefaultIcon : TopicSignedDefaultIcon;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->